docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Distributed_Timers.adoc

```diff
@@ -0,0 +1,52 @@
+[[Jakarta_Enterprise_Beans_Distributed_Timers]]
+= Jakarta Enterprise Beans Distributed Timers
+
+[abstract]
+
+WildFly now supports distributed timers backed by an embedded Infinispan cache.
+The Infinispan cache not only provides an HA solution for persistent timer metadata, in the case of a server crash or restart,
+but also distributes timer execution evenly between cluster members.
+Consequently, this solution generally provides better performance and scalability than the existing database solution.
+
+This feature can also be used to provide file, database, or persistence to a remote Infinispan cluster for single server architectures, effectively replacing the existing file-store and database-store TimerService facilities.
```

Comment:
Not sure what this refers to: "can be used to provide file, database or persistence ??? to a remote Infinispan cluster". Is the word timers or similar missing?

Reply:
Yeah - that sentence doesn't make sense.  It should read:
> This feature can also be used to provide file, database, or remote Infinispan cluster persistence of timer metadata for single server architectures,  ...